### PR TITLE
i18n(es): Updated `cloudflare.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `cloudflare.mdx` based on this commit 9d026521bc4ef73e817d72464df3ece3b68715cd

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/cloudflare.mdx [hunk 1]
```diff
@@ -58,7 +58,11 @@ Cloudflare Pages tiene 2 modos diferentes para implementar funciones, el modo `a
...
 
-En el modo `directory`, el adaptador compilará la parte del cliente de tu aplicación de la misma manera, pero moverá el script del worker a una carpeta `functions` en la raíz del proyecto. El adaptador solo colocará un archivo `[[ruta]].js` en esa carpeta, lo que te permite agregar plugins adicionales y middleware de páginas que se pueden incluir en el control de versiones. La documentación de Cloudflare contiene más información sobre [cómo escribir funciones personalizadas](https://developers.cloudflare.com/pages/platform/functions/).
+En modo directorio, el adaptador compilará la parte del lado del cliente de tu aplicación de la misma manera de forma predeterminada, pero mueve el script del worker a una carpeta `functions` en la raíz del proyecto. En este caso, el adaptador solo colocará un `[[path]].js` en esa carpeta, lo que te permite agregar complementos adicionales y middleware de páginas que se pueden verificar en el control de versiones.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/cloudflare.mdx:61]:
```suggestion
En modo directorio, el adaptador compilará la parte del lado del cliente de tu aplicación de la misma manera de forma predeterminada, pero mueve el script del worker a una carpeta `functions` en la raíz del proyecto. En este caso, el adaptador solo colocará un archivo `[[path]].js` en esa carpeta, lo que te permite agregar plugins adicionales y middleware de páginas que se pueden verificar en el control de versiones.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Comment by dreyfus92** [comment]
Muchas gracias por el review @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps

**Pull request merged**